### PR TITLE
various: add licenses

### PR DESCRIPTION
**Pull request by miccal**
Helping out on https://github.com/Homebrew/homebrew-core/issues/58225

Ping @SeekingMeaning @chenrui333 @Rylan12

### CODE REVIEW

File: Formula/libomp.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Libomp < Formula
   homepage "https://openmp.llvm.org/"
   url "https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/openmp-10.0.0.src.tar.xz"
   sha256 "3b9ff29a45d0509a1e9667a0feb43538ef402ea8cfc7df3758a01f20df08adfa"
+  license "MIT"
```
Comment (chenrui333) [inline Formula/libomp.rb:6]: Can you show me where did you see `MIT` license reference?

**Comment by chenrui333** [comment]
cancelled the workflow run as not needed.

File: Formula/libomp.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/libomp.rb:6]: happened to dig around with the llvm project's license recently.

File: Formula/wxmac.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class Wxmac < Formula
   homepage "https://www.wxwidgets.org"
   url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2"
   sha256 "440f6e73cf5afb2cbf9af10cec8da6cdd3d3998d527598a53db87099524ac807"
+  license "wxWindows"
```
Comment (chenrui333) [inline Formula/wxmac.rb:6]: Nice

File: Formula/libomp.rb (hunk 1 above)
Comment (Rylan12) [reply Formula/libomp.rb:6]: There's this line on the [homepage](https://openmp.llvm.org):

> All of the code here is dual licensed under the MIT license and the UIUC License (a BSD-like license). The LICENSE.txt file at the top of the OpenMP project contains the license text and associated patent grants.

File: Formula/wxmac.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/wxmac.rb:6]: Could this be `GPL-2.0`?

Comment (chenrui333) [reply Formula/wxmac.rb:6]: You are right 👍

**Review by chenrui333 — COMMENTED** [review]
👍  lgtm, the only concern is around llvm project license.

File: Formula/nettle.rb [hunk 3]
```diff
@@ -4,6 +4,7 @@ class Nettle < Formula
   url "https://ftp.gnu.org/gnu/nettle/nettle-3.6.tar.gz"
   mirror "https://ftpmirror.gnu.org/nettle/nettle-3.6.tar.gz"
   sha256 "d24c0d0f2abffbc8f4f34dcf114b0f131ec3774895f3555922fe2f40f3d5e3f1"
+  license "GPL-2.0"
```
Comment (Rylan12) [inline Formula/nettle.rb:7]: I think this is `LGPL-3.0`

See https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYING.LESSERv3

File: Formula/libomp.rb (hunk 1 above)
Comment (miccal) [reply Formula/libomp.rb:6]: @Rylan12 is correct -- that is precisely where I got it from.

File: Formula/nettle.rb (hunk 3 above)
Comment (miccal) [reply Formula/nettle.rb:7]: I believe this one is multi-licensed with `GPL-2.0`, `GPL-2.0` and `LGPL-3.0` -- see [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYINGv2) and [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYINGv3) and [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYING.LESSERv3).

According to [ravenports](http://www.ravenports.com/catalog/bucket_4B/nettle/standard/) it is:
```
GPLv2+ , LGPL3+ (dual)
```

Comment (SMillerDev) [reply Formula/nettle.rb:7]: For multi license you should make a comment with the actual licenses and pick one. It needs to be clear what to add once multi license support is ready.

Comment (miccal) [reply Formula/nettle.rb:7]: @SmillerDev do you have an example of the actual comment you would like, for consistency?

Comment (SMillerDev) [reply Formula/nettle.rb:7]: I'm just making something up right now, but how about something like:
`//license ['a', 'b'] - awaiting https://github.com/Homebrew/brew/pull/7953`

Comment (miccal) [reply Formula/nettle.rb:7]: How about:
```
# license ["GPL-2.0", "GPL-3.0", "LGPL-3.0"] -- pending https://github.com/Homebrew/brew/pull/7953
```

Comment (miccal) [reply Formula/nettle.rb:7]: And if it is ok, could this PR be merged and I will then send through another one with the required updates for `nettle`, thanks.

File: Formula/libomp.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/libomp.rb:6]: Sounds good! 👍

File: Formula/nettle.rb (hunk 3 above)
Comment (chenrui333) [reply Formula/nettle.rb:7]: yeah, we can certainly iterate on this process.

Comment (chenrui333) [reply Formula/nettle.rb:7]: If you can apply the same change to your previous PRs, that would be more than welcome, we can batch modify the formulae once the `brew` got merged.

Comment (miccal) [reply Formula/nettle.rb:7]: Thanks @chenrui333.

Comment (chenrui333) [reply Formula/nettle.rb:7]: Thank you for your help! ❤️

**Pull request merged**